packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -252,6 +254,8 @@ async function initializeData() {
 
 	try {
 		await Promise.all(loadPromises);
+
+		collaborationStore.notifyWorkflowOpened(workflowId.value);
```

Comment:
Is this the correct place where this should be sent?

Reply:
Maybe a better place would be the `initializeWorkspaceForExistingWorkflow` or `openWorkflow` method in the same file.